lib/aws-embedded-metrics-customink/config.rb

```diff
@@ -30,9 +32,14 @@ def reconfigure
 
           def namespace
             return @namespace if defined?(@namespace)
+
```

Comment:
This method is two lines long, did it need a paragraph break?

Reply:
I was holding most of my lint tweaks back for the next PR but this one got lumped into the min stageable chunk. It's ruby style to follow any guard clause with an empty line: https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/cop/layout/empty_line_after_guard_clause.rb